draft-ietf-ccwg-rfc5033bis.md

```diff
@@ -248,23 +248,26 @@ check-list.
 explanation of the deviations from {{!RFC2914}}.
 
 (1)
-: Impact on Standard TCP, SCTP {{!RFC2960}}, and DCCP {{!RFC4340}}.
+: Impact on existing deployments of TCP {{!RFC9293}}, SCTP {{!RFC9260}},
+DCCP {{!RFC4340}}, and QUIC {{!RFC9000}}.
 
 : Proposed congestion control mechanisms should be evaluated when
-competing with standard IETF congestion control {{!RFC2581}},
-{{!RFC2960}}, {{!RFC4340}}.  Alternate congestion controllers that have a
-significantly negative impact on traffic using standard
-congestion control may be suspect and this aspect should be part
-of the community's decision making with regards to the
-suitability of the alternate congestion control mechanism.
-
-: We note that this bullet is not a requirement for strict TCP-
+competing with standard IETF congestion control {{!RFC5681}},
+{{!RFC9260}}, {{!RFC4340}}, {{!RFC9002}}, {{!RFC9438}}.  Alternate
+congestion controllers that have a significantly negative impact on
+traffic using standard congestion control may be suspect and this aspect should
+be part of the community's decision making with regards to the suitability of
+the alternate congestion control mechanism. The community should also consider
+any other non-standard congestion controls shown to be widely deployed,
+
+: We note that this bullet is not a requirement for strict Reno- or Cubic-
 friendliness as a prerequisite for an alternate congestion
 control mechanism to advance to Experimental.  As an example,
 HighSpeed TCP is a congestion control mechanism that is
 Experimental, but that is not TCP-friendly in all environments.
 We also note that this guideline does not constrain the fairness
-offered for non-best-effort traffic.
+offered for non-best-effort traffic. Existing major deployments should not
+suffer severe performance degradation when a new algorithm is deployed.
```

Comment:
should not suffer severe performance degradation... for Non-best effort. That I could not parse.

Agree to the idea, but I am not a fan of the words "severe performance degradation" unless we define this, is there a better word for harm or a need to define?

Reply:
I rearranged the sentence to remove the implication the last sentence relates to non-best-effort traffic.

I'm comfortable with applying a "know it when I see it'" standard for severe performance degradation, but if you have other words (or a definition) I'm happy to accept it.

Reply:
Would this be a way to go? "When  a new algorithm is deployed, the existing major deployments need to be considered to avoid severe performance degradation."

Reply:
SGTM